python/mxnet/numpy/multiarray.py

```diff
@@ -6419,3 +6419,39 @@ def einsum(*operands, **kwargs):
     ...     np.einsum('ijk,ilm,njm,nlk,abc->',a,a,a,a,a, optimize=True)
     """
     return _mx_nd_np.einsum(*operands, **kwargs)
+
+@set_module('mxnet.numpy')
+def column_stack(tup):
+    """ column_stack(*args, **kwargs)
```

Comment:
No need to define the signature in docstring.

Reply:
Removed.